recipe-server/client/control_new/state/actions/selectors.js

```diff
@@ -1,5 +1,13 @@
-/* eslint import/prefer-default-export: "off" */
-
 export function getAction(state, id, defaultsTo = null) {
-  return state.app.actions.items.get(id, defaultsTo);
+  // Convert string IDs to integers, since the state is keyed by ints.
```

Comment:
I actually think this conversion business should remain; `getAction` is not always used within a `connect` or with router params. I found these cases after searching:

`getAction(state, revision.getIn(['recipe', 'action_id']));`
`getAction(state, recipe.get('action_id'));`
```@connect(
  (state, props) => ({
    selectedAction: getAction(
      state,
      props.form.getFieldValue('action_id'),
      new Map(),
    ),
  }),
)```

In order to keep these sections concise, it makes sense to me to leave the int parsing within `getAction`.